doc/sphinx/source/project/about.rst

```diff
@@ -1,101 +1,182 @@
 .. about: 
 
-Overview
-========
+
+About Features, FAQs
+####################
+
+* :ref:`features`
+* :ref:`faq`
+* :ref:`whats_next`
 
 
-Welcome to the documentation site for |InG|, an open-source C++ Compiler, 
-Library, and runtime suite for Deep Learning frameworks running training and inference on 
-:abbr:`Deep Neural Network (DNN)` models. nGraph is framework-neutral and can be 
-targeted for programming and deploying :abbr:`Deep Learning (DL)` applications 
-on the most modern compute and edge devices.   
+.. _features:
 
 Features
+========
+
+The nGraph :abbr:`Intermediate Representation (IR)` contains a combination of 
+device-specific and non device-specific optimization and compilations to  
+enable:
+
+* **Fusion** -- Fuse multiple ``ops`` to to decrease memory usage "localities". 
+* **Memory management** -- Prevent peak memory usage by intercepting a graph 
+  with or by a "saved checkpoint," and to enable data auditing. 
+* **Data reuse** -- Save result and reuse for subgraphs with the same input
+* **Graph scheduling** -- Run similar subgraphs in parallel 
+* **Graph partitioning** -- Partition subgraphs to run on different devices to 
+  speed up computation.
+* :abbr:`Direct EXecution mode (DEX)` or **DEX** -- Execute kernels for the 
```

Comment:
I still don't understand why users would care about DEX. It is an implementation detail.

Reply:
Removed